ecpy/instruments/widgets/profile_edition.enaml

```diff
@@ -332,6 +332,20 @@ enamldef RenameSettingsPopup(PopupView): pop:
 _DestroyableContainer = add_destroy_hook(Container)
 
 
+class _Include(Include):
+    """Re-implemented include to show widget after parenting.
+
+    When destroy_old is False the widget is hidden when un-parented and need
+    to be shwon again when re-parented.
+
+    """
+    def _observe_objects(self, change):
+        super(_Include, self)._observe_objects(change)
+        if self.is_initialized and change['value']:
+            for o in self.objects:
```

Comment:
change['value'] contains the new value of the 'objects' member. I could check self.objects instead. I simply take no risk and always all the children.